packages/root-cms/ui/components/DocEditor/fields/ImageField.tsx

```diff
@@ -8,13 +8,24 @@ import {joinClassNames} from '../../../utils/classes.js';
 import {uploadFileToGCS} from '../../../utils/gcs.js';
 import {FieldProps} from './FieldProps.js';
 
-export const IMAGE_MIMETYPES = [
-  'image/png',
+/** Mimetypes accepted by the image input field. */
+const IMAGE_MIMETYPES = [
```

Comment:
Only keep `images/gif` here, for all the others we don't want to encourage use of them by default. Also image/jpg is not a valid mimetype so please remove.

Reply:
Done